data-prepper-plugins/prometheus-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/prometheus/configuration/AuthenticationOptions.java

```diff
@@ -8,10 +8,10 @@
 
 public class AuthenticationOptions {
 
-    @JsonProperty("http_basic")
+    @JsonProperty("http-basic")
```

Comment:
Please change this back to `http_basic`.

Reply:
@dlvenable Resolved